datadog/resource_datadog_synthetics_test_.go

```diff
@@ -466,20 +562,48 @@ func updateSyntheticsTestLocalState(d *schema.ResourceData, syntheticsTest *data
 	d.Set("request_headers", actualRequest.Headers)
 
 	actualAssertions := syntheticsTest.GetConfig().Assertions
-	var localAssertions []map[string]string
```

Comment:
Do we need to keep any of the existing logic for users that still use `assertions` for a bit?

Reply:
Yeah I think I found what was needed there.